### PR TITLE
Correct documentation in GestureRecognizerState

### PR DESCRIPTION
**Pull request by tvolkert**
## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tvolkert** [comment]
@Hixie looking for a test exemption please

File: packages/flutter/lib/src/gestures/recognizer.dart [hunk 1]
```diff
@@ -355,8 +355,15 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
 /// The possible states of a [PrimaryPointerGestureRecognizer].
 ///
 /// The recognizer advances from [ready] to [possible] when it starts tracking a
-/// primary pointer. When the primary pointer is resolved in the gesture
-/// arena (either accepted or rejected), the recognizers advances to [defunct].
+/// primary pointer. Where it advances from there depends on how the gesture is
+/// resolved for that pointer:
+///
+///  * If the primary pointer is resolved by the gesture winning the arena, the
+///    state remains as [possible] as long as the recognizer continues to track
```
Comment (Hixie) [inline packages/flutter/lib/src/gestures/recognizer.dart:362]: "remains as [possible] as long as" is confusing wording. Maybe "stays in the [possible] state for as long as" or something?

**Comment by Hixie** [comment]
test-exempt: documentation

**Review by Hixie — APPROVED** [review]
LGTM

**Pull request merged**